resident-default.properties

```diff
@@ -573,3 +573,9 @@ mosip.resident.update.service.status.job.enabled=true
 mosip.resident.update.service.status.job.initial-delay=60000
 mosip.resident.update.service.status.job.interval.millisecs=60000
 mosip.resident.grievance=/mock/external/grievance/redressel?name={name}&emailId={email}&phoneNo={phone}&eventId={eventId}
+
+#template required properties
+resident.template.date.pattern=dd-MM-yyyy
+resident.template.time.pattern=HH:mm:ss
+resident.ui.track-service-request-url=https://resident.dev1.mosip.net/#/uinservices/trackservicerequest/
```

Comment:
Take the resident UI base url from the existing url. Don't redefine again.

Reply:
yes done